Vagrantfile

```diff
@@ -25,8 +25,6 @@ cluster_ip_nodes = ""
 provision_common_once = <<SCRIPT
 ## setup the environment file. Export the env-vars passed as args to 'vagrant up'
 echo Args passed: [[ $@ ]]
-echo -n "$1" > /etc/hostname
```

Comment:
Can we remove the node_name parameter to provision_common_once at the caller site and I suppose shift the rest of the arguments here?